src/main/java/com/taboola/backstage/internal/config/SerializationConfig.java

```diff
@@ -0,0 +1,64 @@
+package com.taboola.backstage.internal.config;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SerializationConfig {
+    private final Map<Class<?>, Class<?>> mixins;
+    private final boolean shouldIgnoreAnySetterAnnotation;
+
+    private SerializationConfig(Map<Class<?>, Class<?>> mixins, boolean shouldIgnoreAnySetterAnnotation) {
+        this.mixins = mixins;
+        this.shouldIgnoreAnySetterAnnotation = shouldIgnoreAnySetterAnnotation;
+    }
+
+    public Map<Class<?>, Class<?>> getMixins() {
+        return mixins;
+    }
+
+    public boolean shouldIgnoreAnySetterAnnotation() {
+        return shouldIgnoreAnySetterAnnotation;
+    }
+
+    @Override
+    public String toString() {
+        return "SerializationConfig{" +
+                "mixins=" + mixins +
+                ", shouldIgnoreAnySetterAnnotation=" + shouldIgnoreAnySetterAnnotation +
+                '}';
+    }
+
+    public static SerializationConfigBuilder builder() {
```

Comment:
use builder setters instead builder. Since we have only 2 fields here builder class it seems over kill